jsk_fetch_robot/README.md

```diff
@@ -119,7 +119,32 @@ fetch-interface function APIs
   ```
 (send *ri* :stop-grasp)
   ```
+  
+gazebo内で、逆運動学でfetchを動かす。
```

Comment:
@708yamaguchi OK, next please could you make it one commit by `git rebase`, and please add clear commit message (see https://github.com/jsk-ros-pkg/jsk_common/blob/master/CONTRIBUTION.md and commit messages at other pull requests)